onyo/commands/edit.py

```diff
@@ -4,40 +4,43 @@
 import os
 import sys
 
-from git import Repo
+from pathlib import Path
 
 from onyo.utils import (
     build_git_add_cmd,
-    run_cmd,
-    edit_file
+    edit_file,
+    get_editor,
+    get_list_of_assets,
+    run_cmd
 )
 
 logging.basicConfig()
 logger = logging.getLogger('onyo')
 
 
 def build_commit_cmd(files, onyo_root):
-    return ["git -C \"" + onyo_root + "\" commit -m", "edit asset(s).\n\n" + "\n".join(files)]
+    return [f'git -C "{onyo_root}" commit -m "edit asset(s).\n\n"', "\n".join(files)]
 
 
 def prepare_arguments(sources, onyo_root):
     problem_str = ""
     list_of_sources = []
+    asset_list = get_list_of_assets(onyo_root)
+
     if isinstance(sources, str):
         sources = ["".join(sources)]
```

Comment:
I'm confused here. Perhaps I'm misunderstanding, but it looks like it wants to make sure that `sources` is a list. If so, why the `join`? Why not simply:
```
sources = [sources]
```